packages/atomic/cypress/integration/result-list/result-components/result-date.cypress.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  generateComponentHTML,
+  TagProps,
+  TestFixture,
+} from '../../../fixtures/test-fixture';
+import {addResultList} from '../result-list-actions';
+import {
+  resultDateComponent,
+  ResultDateSelectors,
+} from './result-date-selectors';
+
+interface ResultDateProps {
+  field?: string;
+  format?: string;
+}
+
+const addResultDateInResultList = (props: ResultDateProps = {}) =>
+  addResultList({
+    bottomMetadata: generateComponentHTML(
+      resultDateComponent,
+      props as TagProps
+    ),
+  });
+
+describe('Result Date Component', () => {
+  describe('when not used inside a result template', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .withElement(generateComponentHTML(resultDateComponent))
+        .init();
+    });
+
+    it.skip('should remove the component from the DOM', () => {
+      cy.get(resultDateComponent).should('not.exist');
+    });
+
+    it.skip('should log a console error', () => {
```

Comment:
Should certain tests in this PR be skipped? I see ~10 in total,

Reply:
They are skipped because they don't pass yet, I'll have to fix those issues in separate PRs.

Reply:
maybe next time, if we add skip to any tests, we should add next to it right away the 'pending' Jira that we will need to work on 🤔 ?